src/pages/index.js

```diff
@@ -33,30 +49,73 @@ const addProfileModal = document.querySelector("#add-modal");
 const addForm = addProfileModal.querySelector("#add-form");
 const addFormValidator = new FormValidator(validationConfig, addForm);
 addFormValidator.enableValidation();
+const avatarForm = document.querySelector("#avatar-form");
+const avatarFormValidator = new FormValidator(validationConfig, avatarForm);
+avatarFormValidator.enableValidation();
+
+// User Info
+const user = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
+
+let userId;
+api.getUserInfo().then((data) => {
+  const { name, about, avatar } = data;
+  user.setUserInfo({ name, about });
+  user.setAvatar(avatar);
+  userId = data._id;
+});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**